src/webapp/pages/form-page/useForm.ts

```diff
@@ -67,9 +68,18 @@ export function useForm(formType: FormType, id?: Id): State {
     const [isLoading, setIsLoading] = useState(false);
     const currentEventTracker = getCurrentEventTracker();
     const { existingEventTrackerTypes } = useExistingEventTrackerTypes();
+    const { dataPerformanceOverview } = usePerformanceOverview();
     useCheckWritePermission(formType);
     const snackbar = useSnackbar();
 
+    const allDataPerformanceEvents = dataPerformanceOverview?.map(
+        event => event.hazardType || event.suspectedDisease
+    );
+    const existingEventTrackers =
```

Comment:
why is this required? useExistingEventTrackerTypes hook should be getiing the dataPerformanceOverview already. Is there some scenario this does not work ?